### PR TITLE
saved pay methods in sign-in response (fixes #45)

### LINKED ISSUE #45: expose saved payment methods to auth endpoint
To save the UI another HTTP request, let's expose saved payment methods (if the user has them) in the response of the auth login.

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: payments_service/auth/views.py [hunk 1]
```diff
@@ -45,6 +45,12 @@ def post(self, request):
         request.session['buyer_pk'] = buyer['resource_pk']
         request.session['buyer_uuid'] = buyer['uuid']
 
+        # As a convenience, put any saved payment methods in the response
+        # if the user has them.
+        pay_methods = api.braintree.mozilla.paymethod.get(
+            active=1, braintree_buyer__buyer__uuid=buyer['uuid'])
```
Comment (andymckay) [inline payments_service/auth/views.py]: Does True work? I can't remember how DRF parses those, feels more Pythonic :)

**Comment by andymckay** [comment]
r+wc

File: payments_service/auth/views.py (hunk 1 above)
Comment (kumar303) [inline payments_service/auth/views.py]: heh, I was scared to try. Let me see if it works. Hopefully slumber does the right thing.

Comment (kumar303) [inline payments_service/auth/views.py]: yeah, `active=True` on the query string works. So slumber does nothing but DRF handles it.

**Pull request merged**